ddtrace/internal/telemetry/writer.py

```diff
@@ -85,7 +89,7 @@ def _flush_integrations_queue(self):
             self._integrations_queue = []
         return integrations
 
-    def _flush_events_queue(self):
+    def flush_events_queue(self):
```

Comment:
I made flush_events_queue "public" to use it in a test. It seems like a helpful function to expose. I could revert this change since it's unreleased to this fix.

Reply:
If this is unrelated, then I'd leave it out. Can we perhaps trigger this behaviour by calling `disable` instead?

Reply:
I updated the enable/disable tests so they do not depend on internal logic.